README.md

```diff
@@ -209,27 +209,64 @@ Authorization in `root:redhat-appstudio` and `root:redhat-hacbs` workspaces in C
 
 For access to the OpenShift staging cluster, the user must be added to the `stage` team in the `redhat-appstudio-sre` Github organization.
 
-## Enabling Monitoring for workload cluster
+## Monitoring for workload clusters
 
-Prometheus Operator is deployed through ArgoCD. This is configure to run in `appstudio-workload-monitoring` namespace. 
-This prometheus is accessed through exported route, which is secured by github oauth. Github oauth secrets are configured in the cluster through secret
+Note:
 
-`prometheus-proxy-config`
+This section refers to "Grafana cluster" and "Prometheus cluster" as the clusters on which Grafana and Prometheus are deployed, respectively. In a multi-cluster topology, there will be a single cluster on which Grafana is deployed, whereas Prometheus will be deployed on all clusters where collecting metrics is needed.
+
+### Setup
+
+First, create the `appstudio-workload-monitoring` namespace if it does not exist yet:
+
+```
+$ oc create namespace appstudio-workload-monitoring
+```
```

Comment:
Do you want them to run this command, or
```
$ ./hack/setup-monitoring.sh  namespace
```

Reply:
I initially had a function for this in the script, but then I thought it was maybe not needed, for such a simple command. However, I can restore it if it makes things easier. Also, IIRC @bamachrn said that the namespace would be created by ArgoCD? (I haven't verified that yet, though)

Reply:
would you have any pointers to this, please?

Reply:
It's actually automatically creating the namespaces when installing the operators (using the CreateNamespace=true directive), rather than explicitly creating the namespaces.  Here's an example: https://github.com/redhat-appstudio/infra-deployments/blob/4f7edc301a7d01d7710cb75aff7b2c41a1c203f1/argo-cd-apps/base/has.yaml#L28

Reply:
You're right that it's a simple command, but if we can automate creating the namespace, even better.  I just don't have sample code handy for that.  I wouldn't put a ton of effort into automating this one line, since we're already doing this semi-manually.

Reply:
ok, I suggest we address that in a separate PR, once we have a `./components/monitoring/kustomize.yaml` in place (it's currently missing after the `main` branch was switch to pre-KCP)